### PR TITLE
two nits for kubelet syncPod

### PR DESCRIPTION
**Pull request by 249043822**
a useless ‘(’ and a log level should be info

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/30429)

### CODE REVIEW

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -1612,7 +1612,7 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 			if mirrorPod.DeletionTimestamp != nil || !kl.podManager.IsMirrorPodOf(mirrorPod, pod) {
 				// The mirror pod is semantically different from the static pod. Remove
 				// it. The mirror pod will get recreated later.
-				glog.Errorf("Deleting mirror pod %q because it is outdated", format.Pod(mirrorPod))
+				glog.V(3).Infof("Deleting mirror pod %q because it is outdated", format.Pod(mirrorPod))
```
Comment (dchen1107) [inline pkg/kubelet/kubelet.go]: Can we at least keep this at Warning?

**Comment by 249043822** [comment]
@dchen1107 , ok, using warining, please review again, thanks

**Comment by jfrazelle** [comment]
@k8s-bot ok to test

**Comment by 249043822** [comment]
Hi @jfrazelle , it's green now, need lgtm

**Pull request merged**